etna/analysis/outliers/confidence_interval_outliers.py

```diff
@@ -14,10 +14,34 @@
     from etna.models import SARIMAXModel
 
 
+def create_ts_by_column(ts: "TSDataset", column: str) -> "TSDataset":
```

Comment:
Why not to add this function as method to TSDataset or to utils.py module?

Reply:
It seems like this function is closely related to this particular task and don't needed in other places.